src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -47,6 +47,10 @@ class PropertyAccessor implements PropertyAccessorInterface
     /** @var int Allow magic __call methods */
     public const MAGIC_CALL = ReflectionExtractor::ALLOW_MAGIC_CALL;
 
+    public const DO_NOT_BYPASS_PROPERTY_HOOKS = 0;
```

Comment:
```suggestion
    public const BYPASS_PROPERTY_HOOK_NONE = 0;
```
So we can use the pseudo type `PropertyAccessor::BYPASS_PROPERTY_HOOK_*`